service/slice_config_webhook_validation.go

```diff
@@ -159,6 +163,35 @@ func validateRenewNowInSliceConfig(ctx context.Context, sliceConfig *controllerv
 	return nil
 }
 
+func validateRotationIntervalInSliceConfig(ctx context.Context, sliceConfig *controllerv1alpha1.SliceConfig, old runtime.Object) (*controllerv1alpha1.VpnKeyRotation, *field.Error) {
+	oldSliceConfig := old.(*controllerv1alpha1.SliceConfig)
+	// nochange detected
+	if sliceConfig.Spec.RotationInterval == oldSliceConfig.Spec.RotationInterval {
+		return nil, nil
+	}
+	// change detected
+	vpnKeyRotation := controllerv1alpha1.VpnKeyRotation{}
+	exists, _ := util.GetResourceIfExist(ctx, types.NamespacedName{
+		Namespace: sliceConfig.Namespace,
+		Name:      sliceConfig.Name,
+	}, &vpnKeyRotation)
+	if exists {
+		vpnKeyRotation.Spec.RotationInterval = sliceConfig.Spec.RotationInterval
```

Comment:
Do we need to  check for the valid of min and max or will be taken care please add the tests for the same

Reply:
should already be handled!